README.md

```diff
@@ -1,14 +1,61 @@
-
-Example: https://blazor.syncfusion.com/demos/chips/default-functionalities?theme=bootstrap5 
-
-Documentation: https://blazor.syncfusion.com/documentation/chip/getting-started 
-
+The [Blazor Chips](https://www.syncfusion.com/blazor-components/blazor-chips?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples) is a feature-rich component that provides small blocks of text information. A quick start Blazor project that shows how to create and configure the Syncfusion Blazor Chips component in a Blazor WebAssembly app. This project also includes a code snippet to add different types of Blazor Chips, add icons chip and how to customize a chip using styles, letters, and avatars and finally, how to bind a click event to the Blazor Chips component.
 
 ## Project pre-requisites
 Make sure that you have the compatible versions of Visual Studio 2019 or 2022 and .NET Core SDK latest version in your machine before starting to work on this project.
 
 ## How to run this application?
 To run this application, you need to first clone the how-to-add-a-blazor-chip-component repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
+
+## Features and benefits
+
+## Action Chips
+
+Blazor Chips are compact elements for representing simple content and can also trigger actions when selected.
+
+## Choice Chips
+
+Blazor Chips can be grouped together as a collection of choices. Choice chips allow users to select from available options.
+
+## Filter Chips
+
+Filter chips are a collection of Blazor chips that allow users to filter the view items based on a specified parameter.
+
+## Input Chips
+
+Input chips allow Blazor chips to be added or removed. A chips can be created from text provided by the user in an input box or deleted as needed.
+
+## Related links
+[Learn More about Blazor Chips](https://www.syncfusion.com/blazor-components/blazor-chips?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/chip/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
+
+[View Demos](https://blazor.syncfusion.com/demos/chips/default-functionalities?theme=bootstrap5?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples)
```

Comment:
change the URL like this https://blazor.syncfusion.com/demos/chips/default-functionalities?theme=bootstrap5&utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples

Reply:
resolved